src/nvim/os/fs.c

```diff
@@ -148,11 +148,7 @@ int os_nodetype(const char *name)
     case UV_TCP:         // unix only
     case UV_UNKNOWN_HANDLE:
     default:
-#ifdef WIN32
-      nodetype = NODE_OTHER;
```

Comment:
Its not clear to me whether the original [mch_nodetype](https://github.com/vim/vim/blob/4445f7ee708f1a1304526a5979c9dd9883a92a0a/src/os_win32.c#L3510) would return NODE_OTHER or not. The docs don't mention what happens if  `GetFileType(ERROR_FILE_NOT_FOUND)`.

Now that I look at it NODE_WRITABLE seems wrong (https://msdn.microsoft.com/en-us/library/windows/desktop/aa364960%28v=vs.85%29.aspx) since its meant for devices not files. So NODE_NORMAL should be used for non existing files.